app/tests/components_tests/test_serializers.py

```diff
@@ -195,6 +195,45 @@
             },
         ],
     },
+    "3ANG": {
+        "name": "Some annotation",
+        "type": "Three-point angle",
+        "angle": [
+            [78.29, -17.14, 76.82],
+            [76.801, -57.62, 84.43],
+            [77.10, -18.97, 115.48],
+        ],
+        "probability": 0.92,
+        "version": {"major": 1, "minor": 0},
+    },
+    "M3AN": {
+        "name": "Some annotations",
+        "type": "Multiple three-point angles",
+        "angles": [
```

Comment:
How did this test pass?

Reply:
I forgot to include the 3-angle types in an enumeration that happened for the pytest-parameterized tests... that was the problem. Fixed!